sqlite-db/src/models.rs

```diff
@@ -901,3 +901,138 @@ impl From<Settlement> for model::Settlement {
         }
     }
 }
+
+#[derive(Copy, Clone, Debug, PartialEq, sqlx::Type)]
+pub enum FeeFlow {
+    LongPaysShort,
+    ShortPaysLong,
+    None,
+}
+
+pub fn into_complete_fee(
```

Comment:
After giving this a quick try: Not possible because `Option` is not defined within the crate so we run into:

```
Only traits defined in the current crate can be implemented for arbitrary types [E0117]
```

I don't think there is a way around this at the moment - if you know one let me know :)

Reply:
oh well, you could probably simplify it to

```
impl From<(FeeFlow, i64) for model::CompleteFee {

}
```

but feel free to leave as is.

Reply:
The `Options` are actually quite annoying - and only needed to be backwards compatible. I think I'll leave it for now and focus on the tests